### PR TITLE
Bump libats version

### PR DESCRIPTION
**Pull request by houcros**

### CODE REVIEW

File: ota-plus-web/project/build.properties [hunk 1]
```diff
@@ -1 +1 @@
-sbt.version=0.13.17
```
Comment (nkly) [inline ota-plus-web/project/build.properties]: Why don't you bump it to the latest?

Comment (houcros) [reply ota-plus-web/project/build.properties]: You mean 1.x? Some services seem to be using 0.13.x while others use 1.x. Don't know what's the story behind that and if it's safe to bump all the services to 1.x, maybe @koshelev @simao?

Comment (koshelev) [reply ota-plus-web/project/build.properties]: I just bumped some of them while working on a service. So feel free to make the change

Comment (simao) [reply ota-plus-web/project/build.properties]: Discussed offline. Some sbt plugins were not available for 1.x, maybe they are now.

Comment (houcros) [reply ota-plus-web/project/build.properties]: I had to make more changes than I expected to move to sbt 1.2.8, so I thought I'd better open a new PR: #1967.

**Pull request merged**